docs/content/python/getting-started/quickstart/index.md

```diff
@@ -0,0 +1,147 @@
+- Who is making the request? (the "actor")
+- What are they trying to do? (the "action")
+- What are they doing it to? (the "resource")
+
+In Oso, these pieces of information are passed to the `{{% exampleGet "isAllowed" %}}` call: `{{% exampleGet "isAllowed" %}}(actor, action, resource)`.
+`{{% exampleGet "isAllowed" %}}` will return `True` or `False`, and your application can choose how to enforce that decision.
+
+That enforcement can happen in the request handler, at the database layer, or in middleware — here, we've chosen to enforce in the request handler.
+Here's {{% exampleGet "example_app" %}} route that displays a page if this user is allowed to read the associated page.
+
+{{% exampleGet "app_code" %}}
+
+
+Oso denies requests unless you explicitly tell it to accept that sort of request.
+You can tell Oso what requests to accept by providing it with a file full of rules, which we call a policy.
+
+## Write an authorization policy
+Here is a typical policy, written in our declarative language, **Polar**.
+It lets any actor with the role `"user"` read a page, but only actors with the role `"admin"` can write to a page.
+
+We can load our example policy from a file with the extension `.polar`.
+
+{{% exampleGet "load_policy" %}}
+
+Here's the authorization.polar file:
+
+```polar
+allow(actor, action, resource) if
+    role_allow(actor, action, resource);
+
+actor_has_role_for_resource(actor, role_name, resource) if
+    role in actor.{{% exampleGet "getroles" %}}() and
+    role_name = role.name and
+    resource = role.resource;
+
+resource(_type: Page, "page", actions, roles) if
+    actions = ["read", "write"] and
+    roles = {
+        user: {
+            permissions: ["read"]
+        },
+        admin: {
+            permissions: ["write"],
+            implies: ["user"]
+        }
+    };
+ ```
+
+The `allow` rule is the top-level rule that we use to say who can do what in our application.
+In this case, we are delegating to Oso's builtin `role_allow` rule which implements all the
+authorization logic for role-based access control based on the data we provide in `actor_has_role_for_resource`
+and `resource`.
+
+An `actor_has_role_for_resource` rule looks up the roles assigned
+to the actor. Each role is parameterized by the `role_name` and the
+`resource` the role is associated with.
+The Oso builtin policy depends on this rule, so this is required.
+
+A `resource` rule governs access to a specific resource in your app — for instance, a page, an object, a route, or a database entry.
+Here, there are two possible actions that can be done to a `Page`: `"read"` and `"write"`.
+Users can read, but not write, to a `Page`.
+Admins are allowed to write to a `Page`.
+
+The line `implies: ["user"]` says that admins can also do anything users are allowed to do; that is, read a `Page`.
+
+There's much more you can do with Polar, including defining parent-child relationships — we're just scratching the surface here.
```

Comment:
Why should I be interested that Polar allows me to define parent-child relationships?

Also I wouldn't characterize it as something that _Polar_ allows me to do — I think it's something that the Oso Roles configuration system allows me to do.

I'm not sure whether this sentence is trying to promote (A) the idea that Oso Roles is great b/c it's built on top of Polar, which is a very flexible language you can use to encode authorization logic way beyond what we've shown in this guide or (B) that there are additional Oso Roles-specific things that we aren't showing you in this guide for brevity's sake.